setup.py

```diff
@@ -42,4 +42,8 @@
         'Programming Language :: Python :: 3 :: Only',
         'Programming Language :: Python :: Implementation :: PyPy',
     ],
+    tests_require=[
```

Comment:
This is deprecated since setuptools 41.5.0 per https://github.com/pypa/setuptools/pull/1878